docs/source/troubleshooting.rst

```diff
@@ -73,7 +73,7 @@ Debug the test ``test_array_simple`` with Questa, using the VHDL toplevel and th
 
 2. Open a new terminal window or tab, and attach GDB to the running process.
 
-  .. code-block:: shell-session
+  .. code-block::
```

Comment:
The shell syntax parsing failed here since we break into a gdb shell half-way through. Highlighting isn't the most important thing, so I just removed it.